src/components/ButtonCreateList/index.jsx

```diff
@@ -10,7 +12,17 @@ function ButtonCreateList() {
     history.push('/list-view');
   };
 
-  return <button onClick={createNewList}>Create a new list</button>;
+  return (
+    <div className="card text-center mt-4 mb-2">
+      <div className="card-body">
+        <img src={cart} alt="shopping cart logo" />
+        <p className="card-text">Tap "Create shopping list" to get started</p>
```

Comment:
I would change this "Tap" word, it is mostly used for mobile, if you are using a desktop or a laptop without touchscreen, you can't tap it.

Reply:
Thank you so much, Indira! Great observation, we'll appreciate it a lot 😄 .

**_Fixed!_**